src/vm/wren_value.h

```diff
@@ -599,6 +602,12 @@ typedef union
   double num;
 } DoubleBits;
 
+// A struct to store objects for garbage collection
+typedef struct
+{
+  Obj* first;
+} GCState;
```

Comment:
I think we can get rid of this. Either:

1. Just pass around a `Obj**`.
2. Add a `Obj* gray` field to WrenVM. That would avoid having to pass it around everywhere. It's a bit weird since the field is only ever used during collection, but it will probably be handy later when we move to incremental.